src/prometheus_alert_rules/grafana_agent_unavailable.rule

```diff
@@ -1,5 +1,6 @@
+# Based on https://awesome-prometheus-alerts.grep.to/rules.html#prometheus-self-monitoring-1
 alert: GrafanaAgentUnavailable
-expr: up{%%juju_topology%%} < 1
+expr: absent(up{})
```

Comment:
Is it possible for `up` to be absent? I thought it's a special one prom creates for every job.
I guess it could be absent if the alert rule was received by prom, but the job spec is invalid/dropped?

Reply:
As far as I tested, when grafana agent is up and running the result of the expression is `0`

![imagen](https://user-images.githubusercontent.com/939888/179054810-ac887b86-4463-4fb0-8664-7dbf7b6e7b8d.png)


However when Grafana Agent disappears the result of the expression is `1`

![imagen](https://user-images.githubusercontent.com/939888/179055067-bf1400af-9475-4778-8a19-838224385831.png)